Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -1538,10 +1538,22 @@ - (void) testWebSocketParseCookie {
         @[@"a1=b1;a2=b2", @[@"a1=b1;a2=b2"]],
         @[@"a1=b1;expires=b2,3", @[@"a1=b1;expires=b2,3"]],
         @[@"a1=b1;a2=b2,a3=b3;a4=b4", @[@"a1=b1;a2=b2", @"a3=b3;a4=b4"]],
-        @[@"a1=b1;Expires=Mon, 29 Feb 2021 20:21:12 GMT;Path=/",
-          @[@"a1=b1;Expires=Mon, 29 Feb 2021 20:21:12 GMT;Path=/"]],
         @[@"a1=b1;expires=b2,3,a2=b2", @[@"a1=b1;expires=b2,3", @"a2=b2"]],
-        @[@"a1=b1;expires=b1,2,a3=b3;Expires=b3,4", @[@"a1=b1;expires=b1,2", @"a3=b3;Expires=b3,4"]],
+        @[@"a1=b1;expires=b1,2,a3=b3;Expires=b3,4",
+          @[@"a1=b1;expires=b1,2", @"a3=b3;Expires=b3,4"]],
+        
+        // RFC 822, updated by RFC 1123
+        @[@"a1=b1;expires=Sun, 06 Nov 1994 08:49:37 GMT;Path=/",
+          @[@"a1=b1;expires=Sun, 06 Nov 1994 08:49:37 GMT;Path=/"]],
+        
+        // RFC 850, obsoleted by RFC 1036
+        @[@"a1=b1;expires=Sunday, 06-Nov-94 08:49:37 GMT;Path=/",
+          @[@"a1=b1;expires=Sunday, 06-Nov-94 08:49:37 GMT;Path=/"]],
```

Comment:
Weird, we actually left this format out.  I don't see the GCLB format in here, is it already included?

Reply:
Above format is similar right? 

```
GCLB=<token>; path=/; HttpOnly; expires=Tue, 22-Nov-2022 07:21:38 GMT
==> 
a1=b1;expires=Sunday, 06-Nov-94 08:49:37 GMT;Path=/"
```

Only difference we are validating is the three different date formats Pasin added as a [comment](https://issues.couchbase.com/browse/CBL-3941?focusedCommentId=644330&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-644330) right?

Reply:
This are the three date formats Pasin mentioned in the [comment ](https://issues.couchbase.com/browse/CBL-3941?focusedCommentId=644330&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-644330)

Reply:
We didn't actually do RFC 850